deploy/eck-kibana/values.yaml

```diff
@@ -0,0 +1,44 @@
+---
+# Default values for eck-kibana.
+# This is a YAML-formatted file.
+
+# Overridable names of the Kibana instance.
+# By default, this is the Release name set for the chart,
+# followed by 'eck-kibana'.
+#
+# nameOverride will override the name of the Chart with the name set here,
+# so nameOverride: testing, would convert to '{{ Release.name }}-testing'
+#
+# nameOverride: "quickstart"
+#
+# fullnameOverride will override both the release name, and the chart name,
+# and will name the Kibana instance exactly what is specified.
+#
+# fullnameOverride: "quickstart"
+
+# Version of Kibana.
+#
+version: 8.2.3
+
+# Labels that will be applied to Kibana.
+#
+labels: {}
+
+# Annotations that will be applied to Kibana.
+#
+annotations: {}
+
+spec:
+  # Count of Kibana instances to create.
+  #
+  count: 1
+
+  # Reference to ECK-managed Elasticsearch instance.
+  #
+  elasticsearchRef:
+    name: quickstart
```

Comment:
Should this default to `eck-elasticsearch` so that it works out-of-the-box.

Reply:
Here's why I initially left it this way.  When installing these charts using the parent `eck-stack` chart, I'm using `fullnameOverride: quickstart` to make these work out of the box, but I'm seeing now that installing Elasticsearch using `eck-elasticsearch` chart, it will be named `eck-elasticsearch`, by default, and if you subsequently install the `eck-kibana` chart, it will fail to work.  I'll get this changed to `eck-elasticsearch`, and adjust the `eck-stack` chart's values to use this as well.  👍

Reply:
well, actually, upon further testing, the default is this:

```
{{ .Release.Name }}-{{ .Chart.Name }}
```

So installing `eck-elasticsearch` Chart using `eck-elasticsearch` as the chart name, you get:

`eck-elasticsearch-eck-elasticsearch` as the Elasticsearch name.

So you can't make the `eck-elasticsearch` chart, and the `eck-kibana` chart work together out of the box without knowing the release name up front, or using fullnameOverride by default, or not tying the ES/Kibana names to the release name at all.  @pebrc Let me know if you have a preference here.

Reply:
I see the problem. I would in this case at least make it so that it works in the stack chart out of the box. Something like
```yaml
eck-elasticsearch:
  enabled: true
  # This is adjusting the full name here, so the default values of the `eck-kibana` chart will work
  # by default, as it references `eck-elasticsearch` in it's `elasticsearchRef`.
  fullnameOverride: elasticsearch

# If enabled, will use the eck-kibana chart and deploy a Kibana resource.
#
eck-kibana:
  enabled: true
  spec:
    elasticsearchRef:
      name: elasticsearch

Reply:
@pebrc thanks, sounds good.  That was certainly my initial intent when using `fullnameOverride` in the `eck-stack` chart's values files.  I've also added the `elasticsearchRef.name` as requested.  I think the helm unittests are finally working again 🤞 , so this should be good for the final bit of review.